code/ui/manager/src/components/mobile/navigation/MobileNavigation.tsx

```diff
@@ -0,0 +1,82 @@
+import type { FC } from 'react';
+import React, { useState } from 'react';
+import { styled } from '@storybook/theming';
+import { IconButton, Icons } from '@storybook/components';
+import { MobileMenuDrawer } from './MobileMenuDrawer';
+import { MobileAddonsDrawer } from './MobileAddonsDrawer';
+import Panel from '../../../container/Panel';
+
+interface MobileNavigationProps {
+  storyTitle?: string | null | undefined;
+}
+
+export const MobileNavigation: FC<MobileNavigationProps> = ({ storyTitle }) => {
```

Comment:
I suggest we just call `useStorybookApi` directly here instead of passing `storyTitle` down here.

It might make writing stories for this component slightly more complex, but let's try.